operate/services/service.py

```diff
@@ -320,13 +320,19 @@ def generate_config_tendermint(self) -> "HostDeploymentGenerator":
         tendermint_executable = str(
             Path(os.path.dirname(sys.executable)) / "tendermint"
         )
-        tendermint_executable = "tendermint"
+
         if platform.system() == "Windows":
             env["PATH"] = os.path.dirname(sys.executable) + ";" + os.environ["PATH"]
             tendermint_executable = str(
                 Path(os.path.dirname(sys.executable)) / "tendermint.exe"
             )
-            tendermint_executable = "tendermint.exe"
+
+        if not (getattr(sys, "frozen", False) and hasattr(sys, "_MEIPASS")):
```

Comment:
Please add a comment "This condition checks if we are running inside Pyinstaller", etc.